lib/spdy-transport/stream.js

```diff
@@ -25,6 +25,7 @@ function Stream (connection, options) {
   this.path = options.path
   this.host = options.host
   this.headers = options.headers || {}
+  this.headersSent = false
```

Comment:
Add `this.trailers = null` here to make it clearer that trailers are tracked and treated specifically.